apps/api/src/domain/models/tag/commands/create-tag/create-tag.command-handler.ts

```diff
@@ -46,4 +46,12 @@ export class CreateTagCommandHandler extends BaseCreateCommandHandler<Tag> {
     protected buildEvent(command: CreateTag, eventId: string, userId: string): BaseEvent {
         return new TagCreated(command, eventId, userId);
     }
+
+    protected override async persist(
```

Comment:
Let's not do this. Let's do it in the base handler or better yet the repository layer instead.

Reply:
I've removed it for now. [Here](https://www.pivotaltracker.com/story/show/186539279) is a story.